Gemfile.lock

```diff
@@ -443,10 +465,8 @@ DEPENDENCIES
   uglifier (>= 1.3.0)
   useragent
   warden_omniauth
+  webmock
   wicked
 
-RUBY VERSION
-   ruby 2.2.2p95
-
 BUNDLED WITH
-   1.12.5
+   1.11.2
```

Comment:
We should remove this bundled with line and it's value. That way we won't need to update it each time.